python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -1022,7 +1022,7 @@ def test_concat_empty_series(ser_1, ser_2):
     got = gd.concat([Series(ser_1), Series(ser_2)])
     expect = pd.concat([ser_1, ser_2])
 
-    assert_eq(got, expect)
+    assert_eq(got, expect, check_dtype=False)
```

Comment:
Under what conditions do we get a result with different `dtype` than Pandas? It's probably OK that we do, but it's worth documenting.

Reply:
I noticed that this was the same workaround that was done for trying to concatenate an empty and non-empty dataframe. I'll look into this and see if I can list out specific cases though!

Reply:
After adding my changes the tests seemed to be passing without the 'check_dtype=False' argument, so I removed it in my last commit. However, I was getting this warning whenever I ran my tests;

'DeprecationWarning: The default dtype for empty Series will be 'object' instead of 'float64' in a future version. Specify a dtype explicitly to silence this warning. pd.Series([])'

Which is why I'm assuming this condition is there in other tests.

Reply:
Nice! I think we get that warning a lot already. When Pandas changes its behaviour we'll have to match it, but I wouldn't worry about it until then. Marking resolved!

Reply:
Thanks, Ashwin!